From f73407b5e8d9790d6d4c551ff87ead9fa19ecda9 Mon Sep 17 00:00:00 2001
From: mezen <mezen@users.noreply.github.com>
Date: Wed, 14 Aug 2024 12:15:05 +0200
Subject: [PATCH] Renamed regional WDR tv sender

All regional tv sender of WDR are name "WDR HD" in tvg_name.
Using any playlist in hypnotix (Linux Mint) results in that all senders have the same name.
---
 iptv/source.yaml | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/iptv/source.yaml b/iptv/source.yaml
index 705c9d9c..60fb7471 100644
--- a/iptv/source.yaml
+++ b/iptv/source.yaml
@@ -5748,7 +5748,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Aachen HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2018019-b/wdrlz_aachen/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
@@ -5757,7 +5757,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Bonn HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2018021-b/wdrlz_bonn/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
@@ -5766,7 +5766,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Bergisches Land HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2018028-b/wdrlz_wuppertal/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
@@ -5775,7 +5775,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Dortmund HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2018022-b/wdrlz_dortmund/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
@@ -5784,7 +5784,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Düsseldorf HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2018023-b/wdrlz_duesseldorf/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
@@ -5793,7 +5793,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Duisburg HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2018024-b/wdrlz_duisburg/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
@@ -5802,7 +5802,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Köln HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2023550-b/wdrlz_koeln/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
@@ -5811,7 +5811,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Münsterland HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2018025-b/wdrlz_muensterland/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
@@ -5820,7 +5820,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Ostwestfalen HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2018026-b/wdrlz_bielefeld/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
@@ -5829,7 +5829,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Südwestfalen HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2018020-b/wdrlz_siegen/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
@@ -5838,7 +5838,7 @@ tv:
         radio: false
         tvg_id: WDRFernsehen.de
         tvg_logo: https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/wdrhd.png
-        tvg_name: WDR HD
+        tvg_name: WDR Ruhr HD
         url: https://wdrlokalzeit.akamaized.net/hls/live/2018027-b/wdrlz_essen/master.m3u8
       - group_title: IPTV-DE
         group_title_kodi: Regional
